homeassistant/components/homekit_controller/button.py

```diff
@@ -151,6 +154,29 @@ async def async_press(self) -> None:
             await self.async_put_characteristics({key: val})
 
 
+class HomeKitProvisionPreferredThreadCrentials(CharacteristicEntity, ButtonEntity):
```

Comment:
`HomeKitProvisionPreferredThreadCrentials` should be `HomeKitProvisionPreferredThreadCredentials`?